Docs improvements (#3302)
Azure/azure-service-operator: docs/hugo/content/guide/frequently-asked-questions.md

allowing it to adopt the existing resource in Azure) you must manually specify the AzureName
of the `RoleAssignment` as the original UUID. Otherwise, the UUID defaulting algorithm will choose a different UUID since
the namespace has changed.

### How can I import existing Azure resources into ASO?

See [Annotations understood by the operator]({{< relref "annotations#serviceoperatorazurecomreconcile-policy" >}}) for
details about how to control can control whether the operator modifies resources or just watches them.

> There are a few options for actually importing resources:

Reviewer: ```suggestion
There are a few options for importing resources into your cluster:
```

Author: Fixed

* If you're looking to import a large number of Azure resources you can use [asoctl]( {{< relref "tools/asoctl" >}}).

> * If you're looking to import a small number of resources, you can also manually create the resource CRs yourself and apply
>   them. As long as the resource name, type and subscription is the same as the existing Azure resource, ASO will 
>   automatically adopt the resource. Make sure to use the `reconcile-policy` you want.

Reviewer: ```suggestion
* If you're looking to import a small number of resources, you can also manually create the resources in your cluster yourself and apply 
  them. As long as the resource name, type and subscription are the same as the existing Azure resource, ASO will 
  automatically adopt the resource. Make sure to use the `reconcile-policy` you want.
```

Author: Done
